src/connection_string.ts

```diff
@@ -863,9 +886,12 @@ export const OPTIONS = {
   loggerLevel: {
     target: 'logger',
     transform({ values: [value] }) {
-      return new Logger('MongoClient', { loggerLevel: value as LoggerLevel });
+      return new LegacyLogger('MongoClient', { loggerLevel: value as LegacyLoggerLevel });
     }
   },
+  mongoLogger: {
```

Comment:
do we actually want to define this here? it's not actually an option, rather an api provided by the client

Reply:
replaced by `mongoLoggerOptions`